openedx/core/djangoapps/content/course_overviews/tests/test_tasks.py

```diff
@@ -0,0 +1,44 @@
+import mock
+
+from xmodule.modulestore import ModuleStoreEnum
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+
+from ..tasks import enqueue_async_course_overview_update_tasks
+
+
+class GenerateCourseOverviewTests(ModuleStoreTestCase):
+    """
+    Tests the basic wiring of the enqueue_async_course_overview_update_tasks function.
+    """
+    def setUp(self):
+        super(GenerateCourseOverviewTests, self).setUp()
+        self.course_1 = CourseFactory.create(default_store=ModuleStoreEnum.Type.mongo)
+        self.course_2 = CourseFactory.create(default_store=ModuleStoreEnum.Type.mongo)
+
+    @mock.patch('openedx.core.djangoapps.content.course_overviews.models.CourseOverview.update_select_courses')
```

Comment:
So, do you see much value in these mocked tests?  Given that the tests in test_generate_course_overview.py already exercise this code - and without mocks?  :)